packages/backend/src/services/people/index.ts

```diff
@@ -9,4 +10,25 @@ const findPerson: FindPerson = (id) =>
 
 const getPeopleCount: GetPeopleCount = () => prisma.person.count();
 
-export { findPeople, findPerson, getPeopleCount };
+const addPerson: AddPerson = (person) => {
+  if (person.mass < 0 || person.height <= 0) {
+    throw new InvalidArgumentException(
+      'Height or mass cannot be a negative number.',
+    );
+  }
+
+  return prisma.person.create({
+    data: {
+      name: person.name,
+      height: person.height,
+      mass: person.mass,
+      birthYear: person.birthYear || undefined,
+      eyeColor: person.eyeColor || undefined,
+      gender: person.gender || undefined,
+      hairColor: person.hairColor || undefined,
+      skinColor: person.skinColor || undefined,
```

Comment:
wont all of these be `undefined` by default when there are no values under these keys?